docs/contributing.md

```diff
@@ -25,6 +25,31 @@ use GitHub pull requests for this purpose. Consult
 [GitHub Help](https://help.github.com/articles/about-pull-requests/) for more
 information on using pull requests.
 
+Unlike most GitHub projects (but like most VCS projects), we care more about the
+contents of commits than about the contents of PRs. We review each commit
```

Comment:
nit: `s/most/many/` GitHub projects? I don't have statistics, but "most" seems too strong.

Reply:
I'd be very surprised if it's not most, but there's no need to be so specific anyway, so I changed to "many" here. I also changed from "most VCS project" to "many VCS project".